com.borgbase.vorta.yaml

```diff
@@ -0,0 +1,89 @@
+app-id: com.borgbase.vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  # Mounting
+  - --talk-name=org.freedesktop.*
+  # Allow access to keyring
+  - --talk-name=org.freedesktop.secrets
+  # SSH
+  - --socket=ssh-auth
+
+build-options:
+  env:
+    MOUNT_FUSE_PATH: ../tmp/
+
+modules:
+  - dependencies/pyqt5.json  
+  - dependencies/python3-setuptools_scm.json
+  - dependencies/python3-appdirs.json
+  - dependencies/python3-apscheduler.json
+  - dependencies/python3-paramiko.json
+  - dependencies/python3-peewee.json
+  - dependencies/python3-psutil.json
+  - dependencies/python3-python-dateutil.json
+  - dependencies/python3-qdarkstyle.json
+  - dependencies/python3-secretstorage.json
+  - dependencies/python3-setuptools.json
+  - dependencies/python3-setuptools_git.json
+  - dependencies/python3-pytest-runner.json
+  - dependencies/openssl.json
+  - dependencies/python3-borgbackup.json
+  - dependencies/python3-pytest.json
+  - dependencies/libfuse.json
+  - dependencies/python3-llfuse.json
+  - dependencies/openssh.json
+
+
+  - name: vorta
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps .
+      - mkdir -p /app/share/metainfo/
```

Comment:
You could just install here instead of mkdir/cp. It would be better if this is part of your python build system.

Reply:
Inside of setup.py? Could you please give me an example of other python flatpaks, so I can adapt their approach?